index.js

```diff
@@ -124,16 +125,20 @@ module.exports = {
 
         //Adding event listeners to take automatic screenshot
 
+        //Adding event listeners to take automatic screenshot
         if (autoCaptureOptions.indexOf('click') !== -1) {
 
-            nemo.driver.flow_.on('scheduleTask', function (task) {
+            flow.on('scheduleTask', function (task) {
+                if (!driver.session_) {
```

Comment:
Would it be more appropriate to instead use ```driver.getSession()``` instead of using what looks like a private variable?